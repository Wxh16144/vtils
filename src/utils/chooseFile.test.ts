import { chooseFile } from './chooseFile'

describe('chooseFile', () => {
  test('ok', () => {
    expect(chooseFile).toBe(chooseFile)
  })
})
