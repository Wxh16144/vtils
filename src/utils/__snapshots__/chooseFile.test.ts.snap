// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`chooseFile 多选其他 1`] = `".doc,.docx__true.txt"`;

exports[`chooseFile 多选图片 1`] = `".jpg,.jpeg,.png,.gif__true.txt"`;

exports[`chooseFile 选择其他 1`] = `".doc,.docx__false.txt"`;

exports[`chooseFile 选择图片 1`] = `".jpg,.jpeg,.png,.gif__false.txt"`;
